### PR TITLE
Improve Test Coverage

### PR DESCRIPTION
**Pull request by svchb**

### CODE REVIEW

File: test/schemes/boundary/dummy_particles/dummy_particles.jl [hunk 1]
```diff
@@ -1,142 +1,188 @@
...
+                                                             smoothing_length,
+                                                             viscosity=viscosity)
+                boundary_system = BoundarySPHSystem(boundary, boundary_model)
+                fluid_system = WeaklyCompressibleSPHSystem(fluid,
+                                                           SummationDensity(),
+                                                           state_equation,
+                                                           smoothing_kernel,
+                                                           smoothing_length)
+
+                # Neighborhood search
```
Comment (efaulhaber) [inline test/schemes/boundary/dummy_particles/dummy_particles.jl:74]:
```suggestion
```
Redundant comment (clear from the variable name).

File: test/schemes/boundary/dummy_particles/dummy_particles.jl [hunk 2]
```diff
@@ -1,142 +1,188 @@
...
-                                                            boundary.coordinates,
-                                                            fluid.coordinates, v_fluid,
-                                                            v_fluid,
-                                                            neighborhood_search)
-
-            for particle in TrixiParticles.eachparticle(boundary_system)
-                if volume[particle] > eps()
-                    TrixiParticles.compute_wall_velocity!(viscosity, boundary_system,
-                                                          boundary.coordinates, particle)
+                scales = [1.0, 0.5, 0.7, 1.8, 67.5]
```
Comment (efaulhaber) [inline test/schemes/boundary/dummy_particles/dummy_particles.jl:121]:
```suggestion
                scale_factors = [1.0, 0.5, 0.7, 1.8, 67.5]
```

**Review by efaulhaber — CHANGES_REQUESTED** [review]

**Review by efaulhaber — APPROVED** [review]

**Comment by efaulhaber** [comment]
@LasNikas do you want to take a look as well before this is merged?

**Comment by svchb** [comment]
@LasNikas Reminder! This waits for over a week now.

**Review by LasNikas — APPROVED** [review]

**Pull request merged**